assets/chat/js/poll.js

```diff
@@ -331,7 +331,9 @@ class ChatPoll {
   pollEndMessage(winner, winnerPercentage) {
     let message = `The poll has ended. Option ${winner} won!`;
     if (winnerPercentage > 0) {
-      message = `The poll has ended. Option ${winner} won with ${winnerPercentage}% of the vote.`;
+      message = `The poll has ended. Option ${winner} won with ${Math.floor(
```

Comment:
Maybe `Math.round()` is a better choice considering that's what's used for the option bar percentages?